src/client/proxy/http.cc

```diff
@@ -313,7 +313,39 @@ bool HTTPMessage::CreateHTTPRequest() {
     // m_ErrorResponse is set in ExtractIncomingRequest
     return false;
   }
-  HandleJumpService();
+
+  if (!IsJumpServiceRequest())
+    {
+      LOG(info) << "HTTPProxyHandler: not a jump service request";
```

Comment:
`info` is for end-user (least technical possible). This looks like a `debug` message.